data/data/aws/bootstrap/main.tf

```diff
@@ -109,7 +109,7 @@ resource "aws_instance" "bootstrap" {
   subnet_id                   = "${var.subnet_id}"
   user_data                   = "${data.ignition_config.redirect.rendered}"
   vpc_security_group_ids      = ["${var.vpc_security_group_ids}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
+  associate_public_ip_address = "true"
```

Comment:
I think we want to drop the quotes, see [here][1].

[1]: https://github.com/openshift/installer/pull/654#discussion_r239206954

Reply:
@wking I thought that I had fixed that already. I'll get that done.

Reply:
Oh, that's a different one. I fixed the one in master.